controllers/storagecluster/external_resources.go

```diff
@@ -344,12 +348,28 @@ func (r *StorageClusterReconciler) createExternalStorageClusterResources(instanc
 				r.Log.Error(err, "Could not create CephFilesystemSubVolumeGroup.", "CephFilesystemSubVolumeGroup", klog.KRef(found.Namespace, found.Name))
 				return err
 			}
+		case "Namespace":
+			if d.Name == radosNamespace {
+				radosNamespaceName = d.Data["radosNamespace"]
+				radosNamespace := &cephv1.CephBlockPoolRadosNamespace{ObjectMeta: objectMeta}
+				mutateFn := func() error {
+					radosNamespace.Spec = cephv1.CephBlockPoolRadosNamespaceSpec{
+						BlockPoolName: d.Data["radosNamespace"],
+					}
+					return nil
+				}
+				_, err := ctrl.CreateOrUpdate(context.TODO(), r.Client, radosNamespace, mutateFn)
```

Comment:
where are we taking care of deletion of rados namespace?

Reply:
where it should be present, with blockpool deletion?

Reply:
@iamniting can you help with ths telling the right pace for the external cluster reconcile

Reply:
I see we do expect that services created by external cluster should be deleted by its own reconcile https://github.com/red-hat-storage/ocs-operator/blob/main/controllers/storagecluster/external_resources.go#L274

So I think the better place would be with cephblockpool reconcile

Reply:
Lets add a owner reference that would be enough.

Reply:
We are good with it as we have already mentioned the objectmeta